.github/workflows/build_linux_jax_wheels.yml

```
@@ -75,11 +83,11 @@ on:
       cloudfront_url:
         description: CloudFront base URL pointing to Python index
         type: string
-        default: "https://d25kgig7rdsyks.cloudfront.net/v2"
+        default: "https://rocm.nightlies.amd.com/v2"
```

```suggestion
        default: "https://rocm.devreleases.amd.com/v2"
```

d25kgig7rdsyks.cloudfront.net is equivalent to rocm.devreleases.amd.com. Defaults should **not** point to nightly releases as this increases the likelihood to accidentally build and push a nightly release while aiming for a dev release.

---

Yes, this code needs to follow https://github.com/ROCm/TheRock/blob/main/docs/development/style_guide.md#use-safe-defaults-for-inputs

Furthermore, https://rocm.nightlies.amd.com/v2/gfx94X-dcgpu/jaxlib/ currently contains dev packages like `jaxlib-0.7.1.dev0+selfbuilt-cp312-cp312-manylinux_2_27_x86_64.whl`. We'll need to yank those.